app/commands/submission/representation/process.rb

```diff
@@ -4,99 +4,25 @@ class Submission::Representation::Process
   initialize_with :tooling_job
 
   def call
```

Comment:
All the code below this is moved into `process_results` below

Reply:
This file is now effectively responsible for taking a tooling job and extracting the relevant data to pass to `processresults`